app/styles/nypr-ui.scss

```diff
@@ -39,3 +40,4 @@ $ember-power-select-trigger-ltr-padding: 0 8px 0 0;
 @import "nypr-ui/popup-menu";
 @import "nypr-ui/share-button";
 @import "nypr-ui/utils";
+@import "nypr-ui/brand-header";
```

Comment:
don't put this in the app manifest. user should import the `brand-header` module explicitly in their apps.